staff/YaninaGomezCucchiararo/cv/styles/mainmobile.css

```diff
@@ -0,0 +1,72 @@
+html, body {
```

Comment:
TIP mobile.css instead of mainmobile.css? mainmobile is too long and difficult to read